### PR TITLE
Added code to exit on invalid argument input

### PR DESCRIPTION
**Pull request by bisenpiyush**
## Changes proposed in this pull request
Added code to exit from snappy-start-all.sh and snappy-status-all.sh scripts on invalid argument input.

## Patch testing
Manual

## ReleaseNotes.txt changes
NA

## Other PRs 
NA

### CODE REVIEW

File: cluster/sbin/snappy-status-all.sh [hunk 1]
```diff
@@ -20,6 +20,11 @@
 # Start all snappy daemons - locator, lead and server on the nodes specified in the
 # conf/locators, conf/leads and conf/servers files repsectively
 
+if [ "$#" -gt 0 ]; then
+  echo "Invalid argument::This shell script doesn't require any agrument."
```
Comment (ashetkar) [inline cluster/sbin/snappy-status-all.sh]: Usage: Run this script without any arguments.

File: cluster/sbin/snappy-start-all.sh [hunk 2]
```diff
@@ -75,6 +75,8 @@ while (( "$#" )); do
       clustermode="rowstore"
     ;;
     *)
+    echo "Invalid argument: $1"
```
Comment (ashetkar) [inline cluster/sbin/snappy-start-all.sh]: Invalid argument: $1
Usage: ./snappy-start-all.sh [-bg | --background | -fg | --foreground]  [-conf conf_dir | --config conf_dir]
 [rowstore]

Comment (bisenpiyush) [reply cluster/sbin/snappy-start-all.sh]: Added usage

File: cluster/sbin/snappy-status-all.sh (hunk 1 above)
Comment (bisenpiyush) [reply cluster/sbin/snappy-status-all.sh]: Resolved

**Pull request merged**